#[test]
fn main(){
    panic!("error: crash and burn");
}